superset-frontend/src/pages/DatasetList/DatasetList.test.tsx

```diff
@@ -197,7 +197,7 @@ describe('DatasetList', () => {
     ).toBeFalsy();
     act(() => {
       wrapper
-        .find('#duplicate-action-tooltop')
+        .find('#duplicate-action-tooltip')
```

Comment:
notable

Reply:
It looks like you got it in the other location where it's set, so we should be fine.